lib/exabgp/application/control.py

```diff
@@ -217,7 +217,7 @@ def fifo_writer (line):
 		}
 
 		def consume (source):
-			if not backlog and b'\n' not in store:
+			if not backlog[source] and b'\n' not in store[source]:
```

Comment:
I fixed this here, as I assume this was wrong, unless you were explicitly looking for \n in store keys :)